README.md

```diff
@@ -1 +1,281 @@
-# math-magicians-app
\ No newline at end of file
+<a name="readme-top">Math Magician App</a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
```

Comment:
- [Optional] Consider removing all the comments in your readme file so that the file can be clean. The comments are just guides on how to fill the Readme appropriately.